CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
         - **.PATCH**: Pull Request Title (PR Author) [PR Number](Link to PR)
 -->
 #### 6.7
+- **.2**: Fix infinity loop (on winvim) in FindParentVCSRoot (Eugenij-W) [#](https://github.com/preservim/nerdtree/pull/1095)
```

Comment:
```suggestion
- **.2**: Fix infinity loop (on winvim) in FindParentVCSRoot (Eugenij-W) [#1095](https://github.com/preservim/nerdtree/pull/1095)
```

You forgot the PR number in the hyperlink.

Reply:
Done, glad to help.